src/playbook.yaml

```diff
@@ -2,6 +2,36 @@
   become: yes
 
   tasks:
-  - name: Print debug message
-    debug:
-      msg: Hello, world!
+
+  - name: Add Docker GPG key
+    apt_key: url=https://download.docker.com/linux/ubuntu/gpg
+
+  - name: Add Docker APT repository
+    apt_repository:
+      repo: deb [arch=amd64] https://download.docker.com/linux/{{ ansible_distribution|lower }} {{ansible_distribution_release}} stable
+      update_cache: yes
+
+  - name: install packages
+    package:
+      name:
+        - docker-ce
+        - docker-ce-cli
+        - containerd.io
+      state: present
+
+  - name: Docker post-install
+    shell: |
+      if [ -z $(getent group docker) ]; then
+        groupadd docker
+      fi
+      gpasswd -a {{ lookup('env','SSH_USERNAME') }} docker
+
+  - name: install python packages
+    pip:
+      name:
+        - awscli==1.18.19
+        - awsmfa==0.2.9
```

Comment:
hmm, wondering if this should go into the base image instead?

Reply:
removing now

Reply:
installed awsmfa in base image https://github.com/Sage-Bionetworks/packer-base-ubuntu-bionic/pull/11